include/capstone/capstone.h

```diff
@@ -199,6 +199,10 @@ typedef enum cs_mode {
 	CS_MODE_RISCV32  = 1 << 0,        ///< RISCV RV32G
 	CS_MODE_RISCV64  = 1 << 1,        ///< RISCV RV64G
 	CS_MODE_RISCVC   = 1 << 2,        ///< RISCV compressed instructure mode
+	CS_MODE_RISCVFD  = 1 << 3,
+	CS_MODE_RISCVV   = 1 << 4,
+	CS_MODE_RISCVZFINX = 1 << 5,
+	CS_MODE_RISCVZCMP_ZCMT_ZCE = 1 << 6,
```

Comment:
I would suggest to add a `_` after RISCV for better readability.
`CS_MODE_RISCV_FD`